syntax.php

```diff
@@ -96,16 +83,22 @@ function handle($match, $state, $pos, Doku_Handler $handler) {
      */
     function render($mode, Doku_Renderer $renderer, $data) {
         global $lang;
+	global $ID;
+	global $INFO;
 
-        if ($mode == 'xhtml') {
+	if ($mode == 'xhtml') {
             $renderer->info['cache'] = false;
 
             @require_once(DOKU_INC.'inc/fulltext.php');
-            $backlinks = ft_backlinks($data[0]);
+	    $page_id = $data[0];
+	    if($page_id == '.') {
+		$page_id = $INFO['id'];
+	    }
```

Comment:
Well, this was some kind of hack. My goal was to make the plugin useable inside a sidebar-page.
Therefore the "current page" symbol "." in context of sidebar-page shoud resolve to the displayed wikipage displayed at runtime. Therefore it should not be cached. So i looked for a way to transport this information to the rendering function and used the dot. Maybe we should extend syntax here for usage in sidebar?